datamover/pkg/sonyoda/odalifecycle.go

```diff
@@ -0,0 +1,77 @@
+package sonyoda
+
+import (
+	"github.com/aws/aws-sdk-go/aws"
+	"github.com/aws/aws-sdk-go/aws/credentials"
+	"github.com/aws/aws-sdk-go/aws/session"
+	"github.com/aws/aws-sdk-go/service/s3"
+	. "github.com/opensds/multi-cloud/datamover/pkg/utils"
+	log "github.com/sirupsen/logrus"
+)
+
+//Sony-ODA only support standard(Tier-1)
```

Comment:
if this is not required/applicable then please remove it. SONY ODA doesn't support different storage class.